### PR TITLE
update the Ubuntu products

### PR DESCRIPTION
**Pull request by konstruktoid**
Signed-off-by: Thomas Sjögren <konstruktoid@users.noreply.github.com>

#### Description:

This PR updates the Ubuntu products
- adds `pkg_manager_config_file: "/etc/apt/apt.conf"`
- corrects `aide_bin_path` since `/usr/bin/aide.wrapper` isn't present in 22.04 and not needed in 20.04

Missing `pkg_manager_config_file`:
image

### CODE REVIEW

**Review by jan-cerny — APPROVED** [review]
I have checked that the rule clean_components_post_updating contains the /etc/apt/apt.conf path with this patch.

The changes LGTM.

@ComplianceAsCode/ubuntu-maintainers PTAL

File: products/ubuntu2004/product.yml [hunk 1]
```diff
@@ -8,14 +8,15 @@ benchmark_root: "../../linux_os/guide"
...
 pkg_manager: "apt_get"
+pkg_manager_config_file: "/etc/apt/apt.conf"
 
 init_system: "systemd"
 oval_feed_url: "https://people.canonical.com/~ubuntu-security/oval/com.ubuntu.focal.cve.oval.xml"
 
 grub2_boot_path: "/boot/grub"
 grub2_uefi_boot_path: "/boot/grub"
 
-aide_bin_path: "/usr/bin/aide.wrapper"
```
Comment (dodys) [inline products/ubuntu2004/product.yml:18]: I believe aide.wrapper still exists in Ubuntu 20.04 and CIS benchmark still explicitly mentions it. I wouldn't change it unless needed. Can you confirm it?

**Review by dodys — COMMENTED** [review]
Just a doubt on 20.04

File: products/ubuntu2004/product.yml (hunk 1 above)
Comment (konstruktoid) [reply products/ubuntu2004/product.yml:18]: It does, and 20.04 "1.3.2 Ensure filesystem integrity is regularly checked" is mentioning `aide.wrapper` unless using timers.
I'll revert that.

**Comment by konstruktoid** [comment]
@dodys - fixed

**Review by dodys — APPROVED** [review]
lgtm, thanks!

**Comment by dodys** [comment]
@jan-cerny can we merge this or do we need to wait for the test to pass? it seems to be stuck since yesterday

**Comment by jan-cerny** [comment]
@dodys Unfortunately I can't because that test is required. I tried to rerun it but for some unknown reason it haven't picked up. I haven't experienced this on other PRs. @konstruktoid Can you please try to rebase the PR on the top of the latest upstream master branch? Hopefully a new push we trigger the CI correctly.

**Comment by konstruktoid** [comment]
Total failure rebase

will reopen

**Pull request closed without merging**